TOOLS/VME4L_API/VME4L_RWEX/vme4l_rwex.c

```diff
@@ -118,7 +118,7 @@ static void usage(int excode)
 	printf("              if fails, store read data in <file>"VERIFY_FILE_POSTFIX"\n");
 	printf("-m            memory map instead of using ioctl calls\n");
 	printf("\n");
-	printf("git version: %s\n", GIT_VERSION);
```

Comment:
I'm against this change

Reply:
I didn't find out where GIT_VERSION is defined, and got compile errors if not commented out.

Reply:
It should come from the build system. Please change the build system accordingly.